components/brave_wallet/browser/zcash/rust/BUILD.gn

```diff
@@ -45,6 +45,62 @@ source_set("orchard_impl") {
   ]
 }
 
+rust_static_library("zcash_protocol") {
+  visibility = [ ":zcash_primitives" ]
+  crate_name = "zcash_protocol"
+  crate_root = "librustzcash/components/zcash_protocol/src/lib.rs"
+  allow_unsafe = true
+  sources = [
+    "librustzcash/components/zcash_protocol/src/consensus.rs",
+    "librustzcash/components/zcash_protocol/src/constants.rs",
+    "librustzcash/components/zcash_protocol/src/constants/mainnet.rs",
+    "librustzcash/components/zcash_protocol/src/constants/regtest.rs",
+    "librustzcash/components/zcash_protocol/src/constants/testnet.rs",
+    "librustzcash/components/zcash_protocol/src/lib.rs",
+    "librustzcash/components/zcash_protocol/src/memo.rs",
+    "librustzcash/components/zcash_protocol/src/value.rs",
+  ]
+  deps = [ "//brave/third_party/rust/memuse/v0_2:lib" ]
+}
+
+rust_static_library("zcash_primitives") {
```

Comment:
I think it would be simpler if we just had a single static lib/Cargo.toml for the files we need from these

Reply:
also this should go in a subdirectory so the BUILD.gn and Cargo.toml are together. The standard pattern would be to put BUILD.gn/Cargo.toml in `components/brave_wallet/browser/zcash/rust/librustzcash`  and then check out `librustzcash` into `components/brave_wallet/browser/zcash/rust/librustzcash/src`

Reply:
Ok, I didn't realize we'd have to change code. In that case it can stay separate